instrumentation/all/opentelemetry-instrumentation-all.gemspec

```diff
@@ -26,43 +26,43 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= #{File.read(File.expand_path('../../gemspecs/RUBY_REQUIREMENT', __dir__))}"
 
-  spec.add_dependency 'opentelemetry-instrumentation-active_model_serializers', '~> 0.22.0'
-  spec.add_dependency 'opentelemetry-instrumentation-anthropic', '~> 0.1.0'
-  spec.add_dependency 'opentelemetry-instrumentation-aws_lambda', '~> 0.4.0'
-  spec.add_dependency 'opentelemetry-instrumentation-aws_sdk', '~> 0.8.0'
-  spec.add_dependency 'opentelemetry-instrumentation-bunny', '~> 0.22.0'
-  spec.add_dependency 'opentelemetry-instrumentation-concurrent_ruby', '~> 0.22.0'
-  spec.add_dependency 'opentelemetry-instrumentation-dalli', '~> 0.27.0'
-  spec.add_dependency 'opentelemetry-instrumentation-delayed_job', '~> 0.23.0'
-  spec.add_dependency 'opentelemetry-instrumentation-ethon', '~> 0.23.0'
-  spec.add_dependency 'opentelemetry-instrumentation-excon', '~> 0.24.0'
-  spec.add_dependency 'opentelemetry-instrumentation-faraday', '~> 0.28.0'
-  spec.add_dependency 'opentelemetry-instrumentation-grape', '~> 0.3.0'
-  spec.add_dependency 'opentelemetry-instrumentation-graphql', '~> 0.29.0'
-  spec.add_dependency 'opentelemetry-instrumentation-grpc', '~> 0.2.0'
-  spec.add_dependency 'opentelemetry-instrumentation-gruf', '~> 0.3.0'
-  spec.add_dependency 'opentelemetry-instrumentation-http', '~> 0.25.0'
-  spec.add_dependency 'opentelemetry-instrumentation-http_client', '~> 0.24.0'
-  spec.add_dependency 'opentelemetry-instrumentation-httpx', '~> 0.3.0'
-  spec.add_dependency 'opentelemetry-instrumentation-koala', '~> 0.21.0'
-  spec.add_dependency 'opentelemetry-instrumentation-lmdb', '~> 0.23.0'
-  spec.add_dependency 'opentelemetry-instrumentation-mongo', '~> 0.23.0'
-  spec.add_dependency 'opentelemetry-instrumentation-mysql2', '~> 0.29.0'
-  spec.add_dependency 'opentelemetry-instrumentation-net_http', '~> 0.24.0'
-  spec.add_dependency 'opentelemetry-instrumentation-pg', '~> 0.30.0'
-  spec.add_dependency 'opentelemetry-instrumentation-que', '~> 0.9.0'
-  spec.add_dependency 'opentelemetry-instrumentation-racecar', '~> 0.4.0'
-  spec.add_dependency 'opentelemetry-instrumentation-rack', '~> 0.27.0'
-  spec.add_dependency 'opentelemetry-instrumentation-rails', '~> 0.37.0'
-  spec.add_dependency 'opentelemetry-instrumentation-rake', '~> 0.3.1'
-  spec.add_dependency 'opentelemetry-instrumentation-rdkafka', '~> 0.7.0'
-  spec.add_dependency 'opentelemetry-instrumentation-redis', '~> 0.26.1'
-  spec.add_dependency 'opentelemetry-instrumentation-resque', '~> 0.6.0'
-  spec.add_dependency 'opentelemetry-instrumentation-restclient', '~> 0.24.0'
-  spec.add_dependency 'opentelemetry-instrumentation-ruby_kafka', '~> 0.22.0'
-  spec.add_dependency 'opentelemetry-instrumentation-sidekiq', '~> 0.26.0'
-  spec.add_dependency 'opentelemetry-instrumentation-sinatra', '~> 0.26.0'
-  spec.add_dependency 'opentelemetry-instrumentation-trilogy', '~> 0.61.0'
+  spec.add_dependency 'opentelemetry-instrumentation-active_model_serializers', '~> 0.22'
```

Comment:
Please update the README explaining the implications of this change. 

Warn users that they may receive unexpected updates of breaking changes from transitive dependencies and they should _pin_ gem versions to avoid that.

Alternatively we can explore a "all-in-one" gem that includes all of the source code in a single gem instead of relying on gem dependencies or add scripts/actions to bump the all gem when a minor version bump of the instrumentations are released. 

That would make it possible to release individual gems and simplify the release process a bit.